composeApp/src/androidMain/kotlin/com/jetbrains/spacetutorial/App.kt

```diff
@@ -0,0 +1,97 @@
+package com.jetbrains.spacetutorial
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.pullrefresh.PullRefreshIndicator
+import androidx.compose.material.pullrefresh.pullRefresh
+import androidx.compose.material.pullrefresh.rememberPullRefreshState
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
```

Comment:
Can we get away with only using `material3` and not `material`?

It looks like there is a [pull to refresh API for Material 3](https://developer.android.com/reference/kotlin/androidx/compose/material3/pulltorefresh/package-summary)